lib/src/main/resources/bag/metadata/afm/2020/afm.xsd

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:dcterms="http://purl.org/dc/terms/"
+    elementFormDefault="qualified"
+    targetNamespace="http://easy.dans.knaw.nl/schemas/bag/metadata/afm/">
+
+    <xs:import namespace="http://www.w3.org/XML/1998/namespace"
+        schemaLocation="http://www.w3.org/2001/03/xml.xsd"/>
+    <xs:import namespace="http://purl.org/dc/terms/"
+        schemaLocation="https://dublincore.org/schemas/xmls/qdc/dcterms.xsd"/>
+
+    <xs:element name="hardware" substitutionGroup="dcterms:description">
+        <xs:annotation>
+            <xs:documentation xml:lang="dut">computertype</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="original_OS" substitutionGroup="dcterms:description">
+        <xs:annotation>
+            <xs:documentation xml:lang="dut">operation system</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+
+    <xs:element name="software" substitutionGroup="dcterms:description">
+        <xs:annotation>
+            <xs:documentation xml:lang="dut">toepassingsprogrammatuur</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="notes" substitutionGroup="dcterms:description">
+        <xs:annotation>
+            <xs:documentation xml:lang="dut"
+                >aanvullingen en specifieke instructies voor
+                (her)gebruik</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="case_quantity" substitutionGroup="dcterms:extent">
+        <xs:annotation>
+            <xs:documentation xml:lang="dut">aantal waarnemingen</xs:documentation>
```

Comment:
VG mentioned some misinterpretation in the past. Shouldn't we elaborate more?

Reply:
that should be mentioned in the conversion-step in easy-fedora2vault